test/DelegateCompensationStaker.t.sol

```diff
@@ -80,3 +80,335 @@ contract InitializeDelegateCompensation is DelegateCompensationStakerTest {
     staker.initializeDelegateCompensation(_delegate);
   }
 }
+
+contract SetAdmin is DelegateCompensationStakerTest {
```

Comment:
Let's put the happy path tests in `DelegateCompensation.t.integration.sol` mainly so we don't mix up what functions implemented on the delegate compensation